UI/SidePanel.cpp

```diff
@@ -2627,13 +2645,13 @@ SidePanel::SidePanel(GG::X x, GG::Y y, GG::Y h) :
     m_system_resource_summary = new MultiIconValueIndicator(Width() - EDGE_PAD*2);
     AttachChild(m_system_resource_summary);
 
-
-    GG::Connect(m_system_name->SelChangedSignal,                        &SidePanel::SystemSelectionChanged, this);
-    GG::Connect(m_button_prev->LeftClickedSignal,                       &SidePanel::PrevButtonClicked,      this);
-    GG::Connect(m_button_next->LeftClickedSignal,                       &SidePanel::NextButtonClicked,      this);
-    GG::Connect(m_planet_panel_container->PlanetSelectedSignal,         &SidePanel::PlanetSelected,         this);
-    GG::Connect(m_planet_panel_container->PlanetRightClickedSignal,     PlanetRightClickedSignal);
-    GG::Connect(m_planet_panel_container->BuildingRightClickedSignal,   BuildingRightClickedSignal);
+    GG::Connect(m_system_name->SelChangedSignal,                         &SidePanel::SystemSelectionChanged, this);
+    GG::Connect(m_button_prev->LeftClickedSignal,                        &SidePanel::PrevButtonClicked,      this);
+    GG::Connect(m_button_next->LeftClickedSignal,                        &SidePanel::NextButtonClicked,      this);
+    GG::Connect(m_planet_panel_container->PlanetSelectedSignal,          &SidePanel::PlanetSelected,         this);
+    GG::Connect(m_planet_panel_container->PlanetLeftDoubleClickedSignal, &SidePanel::PlanetDoubleClicked,    this);
```

Comment:
You don't appear to be doing anything in SidePanel::PlanetDoubleClicked so why not connect directly to the signal as in the following 2 lines?